src/NodeServices/GraphicsDataInterfaces.cs

```diff
@@ -342,6 +342,79 @@ public interface IRenderLabels
         void ClearLabels();
     }
 
+    /// <summary>
+    /// Represents instance matrices and references to tessellated geometry in the RenderPackage
+    /// </summary>
+    public interface IRenderInstances
+    {
+        /// <summary>
+        /// A list of mesh vertices ranges that have associated instance references
+        /// </summary>
+        Dictionary<Guid,Tuple<int, int>> MeshVerticesRangesAssociatedWithInstancing { get; }
```

Comment:
It is an implementation detail mostly but it is also read only.  Because the interaction with render data structures I still need some query mechanism in the public.  I could instead make something like `bool IsBaseTessellationAdded(Guid id)`